owncloudData/src/androidTest/java/com/owncloud/android/data/authentication/datasources/implementation/OCLocalAuthenticationDataSourceTest.kt

```diff
@@ -138,7 +125,7 @@ class OCLocalAuthenticationDataSourceTest {
 
         mockSelectedAccountNameInPreferences()
 
-        ocLocalAuthenticationDataSource.addBasicAccount(
+        val accountName = ocLocalAuthenticationDataSource.addBasicAccount(
             OC_ACCOUNT_ID,
             OC_REDIRECTION_PATH.lastPermanentLocation,
             "password",
```

Comment:
I would use the constant variable OC_BASIC_PASSWORD

Reply:
True! Nice observation 👍